src/rulesets/severity.ts

```diff
@@ -20,40 +20,47 @@ function getSeverityForRule(
   }
 }
 
+function getSeverityForInvalidRule(
+  existingRule: FileRuleSeverity | [FileRuleSeverity] | [FileRuleSeverity, object],
+  newRule: FileRule | FileRulesetSeverity,
+) {
+  if (newRule === 'off') return -1;
```

Comment:
I know this looks insane, but our syntax is so loose, it's difficult to make it any simpler.